src/javascript/core/user.js

```diff
@@ -8,6 +8,7 @@ let store;
 const defaultUserConfig = {
 	storage: 'cookie',
 	name: 'spoor-id',
+	nameOverride: 'spoor-id',
 	value: null,
 	domain: (document.URL.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i)[1].indexOf('ft.com') > -1 ? 'ft.com' : null)
```

Comment:
Should this use `location.hostname` instead of `(document.URL.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i)[1]`?